packages/components-native/src/ActionLabel/ActionLabel.tsx

```diff
@@ -12,9 +12,9 @@ type ActionLabelType = "default" | "cardTitle";
 
 interface ActionLabelProps {
   /**
-   * Text to display
+   * Text to display. Supports strings, numbers, and nested text nodes.
```

Comment:
Im not sure if we need to call out the types when the IDE will already tell us, and we're just extending ReactNode, which includes a bunch of other stuff too like the last values that always make me laugh a bit

```
    type ReactNode =
        | ReactElement
        | string
        | number
        | Iterable<ReactNode>
        | ReactPortal
        | boolean
        | null
        | undefined
        | DO_NOT_USE_OR_YOU_WILL_BE_FIRED_EXPERIMENTAL_REACT_NODES[
            keyof DO_NOT_USE_OR_YOU_WILL_BE_FIRED_EXPERIMENTAL_REACT_NODES
        ];
```

might be fine to leave the types to our type system, and here we just say something like "supports nesting text elements"?